src/coffee/connect/rest.coffee

```diff
@@ -119,7 +119,15 @@ class Rest
   # Throws an {Error} if `access_token` could not be retrieved
   _preRequest: (params, callback) ->
     _req = (retry) =>
-      unless @_options.access_token
+      isTokenExpired = false
+      if @_tokenExpiration
+        expirationTime = Date.now() + (@_tokenExpiration * 1000) -
```

Comment:
Ah now I get what you mean :)

Yeah you're right, I've put it in the wrong place. Thanks for catching this! 👍